code/game/gamemodes/cult/runes.dm

```diff
@@ -508,9 +509,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 /obj/effect/rune/raise_dead/examine(mob/user)
 	..()
 	if(iscultist(user) || user.stat == DEAD)
-		var/revive_number = 0
-		if(GLOB.sacrificed.len)
-			revive_number = GLOB.sacrificed.len - revives_used
+		var/revive_number = LAZYLEN(GLOB.sacrificed) - revives_used
```

Comment:
No, it doesn't, since if you have less than 0 admins are fucking with stuff and displaying how many people you need to sac to get access to revives would be important information then.